workspace.html

```diff
@@ -52,40 +52,74 @@ <h3>See the Space</h3>
               {% include modal-form.html initiative='tour' style='img-button' %}
               <!-- Video Modal-->
               <div id="see-space-video-modal" class="modal fade lgx-modal">
-                <div class="modal-dialog">
-                  <div class="modal-content">
-                    <div class="modal-header">
-                      <button type="button" class="close" data-dismiss="modal" aria-hidden="true">×</button>
-                    </div>
-                    <div class="modal-body" data-embed="0YgGYadr7rk">
-                      <iframe id="modalvideo" src="https://www.youtube.com/embed/0YgGYadr7rk" allowfullscreen></iframe>
-                    </div>
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
```

Comment:
@sgha229 How come we are importing BS again here?

Reply:
That was just bad coding on my part when I was first playing around with BS.